### DISCUSSION TITLE
slider example not work

### DISCUSSION THREAD

**Discussion by vkatochka** [fomantic/Fomantic-UI#1253] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
I copied the example from https://fomantic-ui.com/modules/slider.html, ran it on jsfiddle and the value in the "input" field has not changed! Is this a bug or am I doing something wrong?

https://jsfiddle.net/fayvqw05/

**Comment by dutrieux** [comment, marked as answer]
You need to add callback 'onChange' like that :

https://jsfiddle.net/1ckhep0s/

**Comment by lubber-de** [comment]
See [behaviors](https://fomantic-ui.com/modules/slider.html#behaviors) and callbacks at https://fomantic-ui.com/modules/slider.html#/settings page

**Comment by vkatochka** [comment]
worked, thanks!
